src/Sql/Sql/Database Backup/Cmdlet/SetAzureRmSqlDatabaseBackupShortTermRetentionPolicy.cs

```diff
@@ -34,6 +34,14 @@ public class SetAzureRmSqlDatabaseBackupShortTermRetentionPolicy : AzureSqlDatab
         [ValidateNotNullOrEmpty]
         public int RetentionDays{ get; set; }
 
+        /// <summary>
+        /// Gets or sets differential backup interval hours.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            Position = 4,
```

Comment:
Too many positional parameters can make the command difficult to use. Please remove this line.

Reply:
OK will update in next push.

Reply:
removed.